camelot/admin/action/application_action.py

```diff
@@ -93,6 +94,20 @@ def __init__( self ):
         self.workspace = None
         self.admin = None
     
+    def get_progress_dialog(self):
+        if self.workspace is not None:
+            view = self.workspace.active_view()
+            if view is not None:
+                if view.objectName() == 'dashboard':
```

Comment:
ik zou hier inderdaad de check doen op 'is er een child gevonden vh type QML view'